inference-engine/src/readers/ir_reader/ie_ir_parser.cpp

```diff
@@ -527,14 +527,20 @@ std::shared_ptr<ngraph::Node> V10Parser::createNode(const std::vector<ngraph::Ou
     // Try to create operation from loaded opsets
     if (!ngraphNode && opsets.count(params.version)) {
```

Comment:
Please also remove NodeConvertor and LayerCreator for Constant op

Reply:
LayerCreator and NodeConverter are not yet removed for Const op due to different attributes layout in IR for current functional tests. It will be removed when all tests will support correct Const op representation (around 90 tests have to be rewritten).

Reply:
But without these changes we don't create a Constant operations through visitor API

Reply:
Const op is removed from NodeConvertor and LayerCreator.